docs/data/data-grid/accessibility/FocusManagement.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {
+  DataGrid,
+  GridColumns,
+  GridRenderCellParams,
+  GridRowsProp,
+} from '@mui/x-data-grid';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+
+const CorrectRenderLink = (props: GridRenderCellParams) => (
+  <Box>
+    <Link tabIndex={props.hasFocus ? 0 : -1} href="/#">
+      more info
+    </Link>
+  </Box>
+);
+
+const WrongRenderLink = () => (
+  <Box>
+    <Link href="/#">more info</Link>
+  </Box>
+);
+
+const correctColumns: GridColumns = [
+  { field: 'link', renderCell: CorrectRenderLink, width: 200 },
+];
+const wrongColumns: GridColumns = [
+  { field: 'link', renderCell: WrongRenderLink, width: 200 },
+];
+
+const rows: GridRowsProp = [{ id: 1 }, { id: 2 }, { id: 3 }, { id: 4 }, { id: 5 }];
+
+export default function FocusManagement() {
+  return (
+    <div style={{ width: '100%', display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
Selecting one cell breaks the demo.

![image](https://user-images.githubusercontent.com/42154031/162451621-131ea4b9-d5ab-48d4-8322-a1bd22b7c69e.png)

Maybe split into two demos so the focus reset button can be used:

<img width="200" src="https://user-images.githubusercontent.com/42154031/162451768-de55ae2c-ba02-4b55-9c51-546d3056076f.png" alt="image" style="max-width: 100%;">

Reply:
I'm not very sure if we should keep two grids inside the same demo. The "reset focus" button is very useful to test the accessibility here. If I click it the focus goes to the first grid.